packages/zora/src/Zora.ts

```diff
@@ -151,58 +151,68 @@ export const simulateMint = async (
       transport: http(),
     })
   const from = account ?? DEFAULT_ACCOUNT
+  let _tokenId = tokenId
+  if (tokenId === null || tokenId === undefined) {
+    const nextTokenId = (await _client.readContract({
+      address: contractAddress,
+      abi: ZORA_MINTER_ABI_1155,
+      functionName: 'nextTokenId',
+    })) as BigInt
 
-  if (tokenId !== null && tokenId !== undefined) {
+    _tokenId = Number(nextTokenId) - 1
+  }
+
+  try {
+    const mintArgs = [
+      FIXED_PRICE_SALE_STRATS[mint.chainId],
```

Comment:
@mmackz found out that there could be mutliple sale strat addresses... we don't need to fix in this PR, but will want to address